src/ui/public/vislib/visualizations/point_series.js

```diff
@@ -211,7 +244,7 @@ export default function PointSeriesFactory(Private) {
 
           self.addClipPath(svg, width, height);
 
-          self.stackedData = {};
+          self.stackData(data);
```

Comment:
yeah it stacks the data (it groups series per axis and then calls d3 stack on that)